src/main/java/Duke.java

```diff
@@ -1,10 +1,167 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String GREETING = "Hello! I'm Duke\nWhat can I do for you?";
+    private static final String GOODBYE = "Bye. Hope to see you again soon!";
+    private static final ArrayList<Task> TASKS = new ArrayList<>();
+
+    /**
+     * @param txt text to indent.
+     * @return indented string.
+     */
+    private static String autoIndent(String txt) {
+        return "    " + txt.replace("\n", "\n    ");
+    }
+
+    /**
+     * prints border line.
+     */
+    private static void borderLine() {
```

Comment:
Wouldn't it be more appropriate to name this method as a verb sucha as printBorderLine ?